engine/src/main/java/org/terasology/network/BroadCastServer.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.network;
+
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.Executors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BroadCastServer {
+
+    private static final Logger logger = LoggerFactory.getLogger(BroadCastServer.class);
+    private static final String DISCOVERY_REQUEST= "DISCOVERY_REQUEST";
+    private static final String DISCOVERY_RESPONSE = "DISCOVERY_RESPONSE";
+
+    private static boolean turnOnBroadcast;
+
+    private DatagramSocket receiveSocket;
+    private DatagramSocket sendSocket;
+
+    public BroadCastServer(){
+        turnOnBroadcast=false;
```

Comment:
I'm not sure if this makes sense, as the default value of `turnOnBroadcast` is already false? :-)

Reply:
Ohk I just added it temporarily due to some other error so I will remove this.

Reply:
This was just a temporary thing which I has used for some other error and I had absolutely no reason for using this, will remove it :grin: